app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sign Up</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+  </head>
+  <body class="signup-body">
+    <div class="container py-5">
+      <div class="row justify-content-center">
+        <div class="col-md-6">
+          <div class="card shadow">
+            <div class="card-body">
+            <header class="header-class">
+                <%= link_to root_path, data: { turbo: false }  do %>
+                  <%= inline_svg_tag "chevron_left.svg", class: "chevron" %>
+                <% end %>                  
+              <div class="header-title-wrapper">
+                <h2 class="mb-4 header-title"><i class="fas fa-user-plus"></i> REGISTER</h2>
+              </div>
+              </header>
+
+              <turbo-frame id="signup_form">
+                <%= form_with(model: resource, as: resource_name, url: registration_path(resource_name), html: { class: 'mt-4' }) do |f| %>
+                  <%= render "devise/shared/error_messages", resource: resource %>
+
+                  <div class="mb-3">
```

Comment:
- You have done great so far :+1: :+1: But there is still an issue you need to fix before moving to the next milestone. :+1: 
- [ ] Just as the previous reviewers kept pointing out, you need to make your auth screens look like the given design. This was clearly stated in the requirements. I'll suggest you apply these layouts diligently as it is required in this milestone.

- - Requirement:- 
![Screenshot from 2024-02-16 19-34-00](https://github.com/yusufyusie/budget_application/assets/107632780/8441c82d-00a0-4794-814e-40d97a431b99)

- [ ] Your Design:-
![Selection_068](https://github.com/yusufyusie/budget_application/assets/107632780/03a3fc29-2e96-4c7a-89bc-146f53513ba3)